packages/frontend-2/lib/workspaces/composables/management.ts

```diff
@@ -1,17 +1,34 @@
+import type { RouteLocationNormalized } from '#vue-router'
```

Comment:
Not a blocker, but what's the difference in this and simply `vue-router`?

Reply:
@andrewwallacespeckle Auto-import imported it that way. The hashtag prefix means that there's an import map/alias defined there, maybe it resolves to different packages in CJS & ESM or Browser & Node. In this case it's not very important cause just `vue-router` works as well